volestipy/setup.py

```diff
@@ -1,68 +1,72 @@
-from os.path import join
 from setuptools import setup
+from setuptools import Extension
 from Cython.Build import cythonize
-from setuptools.extension import Extension
+from os.path import join
 import numpy
 
 
-version = "0.2.0"
+version = "0.3.0"
 license='LGPL3',
 packages = ["volestipy"]
 description="volestipy: wrapper for the VolEsti library to sample from convex sets and compute volume."
-author = "Pedro Zuidberg Dos Martires"
-author_email="pedro.zuidbergdosmartires@cs.kuleuven.be"
+author = "Haris Zafeiropoulos"
+author_email="haris-zaf@hcmr.gr"
 name = 'volestipy'
-scripts = []
-url = ""
-download_url = ""
-classifiers = []
 zip_safe=False
 
-
 source_directory_list = ['volestipy', join('volestipy','src')]
 
-
 compiler_args = [
-    "-std=c++11",
-    "-O3",
-    "-lm",
-    "-ldl",
-    "-DBOOST_NO_AUTO_PTR",
+	"-std=c++11",
```

Comment:
is this a tab? please include only spaces tabs are evil ;) (change your settings of your IDE to replace tabs with a sequence of spaces)

Reply:
Done! :)